web_form_sticky_header/views/templates.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <template id="assets_backend" name="web_form_sticky_header assets" inherit_id="web.assets_backend">
+            <xpath expr="." position="inside">
```

Comment:
according to what we discussed today tthis would place it last. equivalent to  expr="//link[@href=/base/static/src/css/modules.css]"   , you are injecting in https://github.com/OCA/OCB/blob/8.0/addons/web/views/webclient_templates.xml#L90 .   looks ok.